horace_core/configuration/@hor_config/hor_config.m

```diff
@@ -221,7 +221,8 @@
 
         function this = set.pixel_page_size(this, val)
             PixelData.validate_mem_alloc(val);
-            config_store.instance().store_config(this, 'pixel_page_size', val);
+            m_chunk_size = floor(val/36);
```

Comment:
Why `36`? Comment

Reply:
That may be so, but until it is, a comment is needed to explain where `36` comes from.